mk/cargo.sh

```diff
@@ -251,16 +251,16 @@ cargo "$@"
 if [ -n "${RING_COVERAGE-}" ]; then
   # Keep in sync with check-symbol-prefixes.sh.
   # Use the host target-libdir, not the target target-libdir.
-  llvm_root="$(rustc +${toolchain} --print target-libdir)/../bin"
+  llvm_root="$(rustc +"${toolchain}" --print target-libdir)/../bin"
```

Comment:
This looks weird. I wonder if there's a better way of doing it. Maybe removing the "outermost"-looking double quotes, similar to the `target_upper=...` line above?

Reply:
Having outher quotes is fine because $() starings a new context where the quoting is processed independently, so no escaping is required. However for consistency, I'll remove the outer quotes as it has not effect on the variable assignment.

Reply:
Resolved by 9a18e6fdb9312615608f6be0f366f25e8b4367ec